include/communicator_quda.h

```diff
@@ -41,6 +41,7 @@ namespace quda
     int (*coords)[QUDA_MAX_DIM];
     int my_rank;
     int my_coords[QUDA_MAX_DIM];
+    int cstar; // number of C* direction as per openQxD convention
```

Comment:
We extended the Topology struct with a cstar member to indicate how many spatial directions have C* boundaries.